odbc-api/src/connection.rs

```diff
@@ -259,6 +259,20 @@ impl<'c> Connection<'c> {
         let name = U16String::from_vec(buf);
         Ok(name.to_string().unwrap())
     }
+
+    /// Fetch the name of the current catalog being used by the connection and store it into the
+    /// provided `buf`.
+    pub fn fetch_current_catalog_name(&self, buf: &mut Vec<u16>) -> Result<(), Error> {
```

Comment:
As the ODBC attribute is just called `CurrentCatalog`, I would prefer dropping the `_name` suffix from the method names. So just `fetch_current_catalog` and `current_catalog`.

Reply:
Sounds good! Updated the branch